docs/development/input-processing.md

```diff
@@ -543,23 +543,25 @@ constructor and by all methods to change the datastructure, be it setters or
 mutators. The integrity needs to be enforced by the datatype itself to make invalid
 data unexpressable.
 
+### Refinery
 
-### Transformation
+The [Refinery library](../../src/Refinery) defines two basic abstractions to be used
+for processing input securely:
 
-The [Transformation-library](../../src/Refinery/Transformation) declares properties of
-`Transformation`s which are understood as structural conversions between different
-shapes and types of data that do not perform side effects. It further aims at
-providing common `Transformations` to be reused throughout the system. The library
-thus is the tool to transform primitive input data into semantically richer
-structures in a declarative way.
+* A **[`Transformation`](../../src/Refinery/Transformation.php)** is a structural
+  conversion between different shapes and types of data. It does not perform
```

Comment:
From where I stand "perform" sounds a little strange in this context. I think "cause" side effects may be more adequate.

Reply:
+1